Assets/HoloToolkit/Input/Scripts/Cursor/Cursor.cs

```diff
@@ -278,16 +273,10 @@ private void TryLoadPointerIfNeeded()
                 // pointer currently registered with FocusManager, we use it.
 
                 Pointer = FocusManager.Instance.TryGetSinglePointer();
-
-                if (Pointer == null)
```

Comment:
Lines 281 to 290 were added during Build to fix/workaround some bugs. @NeerajW should have more insight.